demo/src/components/Image.tsx

```diff
@@ -12,19 +14,26 @@ interface Props extends React.HTMLAttributes<HTMLImageElement> {
 }
 
 export const Image = ({ src, children, className, id, slide, locatable = true }: Props) => {
+  const containerRef = useRef<HTMLDivElement | null>(null);
+  const preview = usePreview();
   const { members, self } = useMembers();
   const { handleSelect } = useElementSelect(id, false);
-  const activeMember = findActiveMember(id, slide, members);
-  const { outlineClasses, stickyLabelClasses } = getOutlineClasses(activeMember);
-  const memberName = getMemberFirstName(activeMember);
-  const label = self?.connectionId === activeMember?.connectionId ? 'You' : memberName;
+  const activeMembers = findActiveMembers(id, slide, members);
+  const locatedByMe = activeMembers.some((member) => member.connectionId === self?.connectionId);
```

Comment:
small thing, but perhaps instead of `located` we should use the word `occupied` ?

Reply:
(I thought at first `located` was a typo for `locked`!)

Reply:
totally agree, you are absolutely right!